.jslintrc

```diff
@@ -0,0 +1,55 @@
+{
+  "predef": [
+    "document",
+    "navigator",
+    "console",
+    "window",
+    "django",
+    "gettext",
+    "windowname_to_id",
+    "jQuery",
+    "WebSocket",
+    "$",
+    "setTimeout",
+    "alert"
+  ],
+
+  "adsafe":     false,
+  "safe":       false,
+
+  "bitwise":    false,
+  "cap":        false,
+  "confusion":  true,
+  "continue":   true,
+  "css":        true,
+  "debug":      false,
+  "eqeq":       false,
+  "es5":        true,
+  "evil":       false,
+  "forin":      false,
+  "fragment":   true,
+  "newcap":     false,
+  "nomen":      true,
+  "on":         false,
+  "plusplus":   false,
+  "regexp":     true,
+  "sloppy":     false,
+  "sub":        false,
+  "undef":      false,
+  "unparam":    true,
+  "vars":       true,
+  "white":      false,
+  "validthis":  true,
+  "strict_mode":true,
+
+  "browser":    false,
+  "devel":      false,
+  "node":       false,
+  "rhino":      false,
+  "widget":     false,
+  "windows":    false,
+  "indent":     2,
```

Comment:
please use 4 spaces indentation and revert the changes to the JS files because that kind of huge change breaks `git blame` (because you appear author of those lines while you are not, so if something breaks somebody may think you are the person to ask why something is not working)